cmd/vcpctl/provision/provision.go

```diff
@@ -60,22 +65,32 @@ var provisionCmd = &cobra.Command{
 }
 
 func AddProvision(cmd *cobra.Command) {
+
 	provisionCmd.Flags().StringVar(&configFile, "config", "", "vsphere cloud provider config file path")
 	provisionCmd.Flags().BoolVar(&oldInTreeConfig, "oldConfig", false, "old int-tree vsphere configuration file, true or false")
 	provisionCmd.Flags().BoolVar(&interactive, "interactive", true, "specify interactive mode (true) as default, set (false) for declarative mode for automation")
 
-	provisionCmd.Flags().StringVar(&vchost, "vchost", "", "specify vCenter IP ")
-	provisionCmd.Flags().StringVar(&vcport, "vcport", "", "specify vCenter Port ")
-	provisionCmd.Flags().StringVar(&vcUser, "vcuser", "", "specify vCenter user ")
-	provisionCmd.Flags().StringVar(&vcPassword, "vcpassword", "", "specify vCenter Password ")
+	provisionCmd.Flags().StringVar(&vchost, "host", "", "specify vCenter IP ")
```

Comment:
Flag descriptions should start with an uppercase letter, and not have a trailing space.

Reply:
done